docs/wiki/framework/cargo-framework.md

```diff
@@ -141,3 +142,20 @@ Note first arg can be a in-game object or a classname of an object type.
  * Example:
  * [object, vehicle] call ace_cargo_fnc_unloadItem
 ```
+
+### 4.6 Remove/ Delete cargo from vehicle 
+
```

Comment:
A possibility for sure, but we should probably decide on (and document) an official workflow for this as I ran into the same issue when I added new cargo functions previously. It would be kind of nice to be able to merge stuff in without having to wait to reduce conflicts and so on.